From 42086d7b56b0274ab20e7a983f85ccfff080aa40 Mon Sep 17 00:00:00 2001
From: Devajith Valaparambil Sreeramaswamy
 <devajith.valaparambil.sreeramaswamy@cern.ch>
Date: Thu, 31 Oct 2024 09:32:16 +0100
Subject: [PATCH 02/52] Revert "Do not install {Clang,Cling}Config.cmake in the
 project lib dir."

This reverts commit d2cc80e4b945ef0b06d6ccf52529734f117e3620.
---
 interpreter/llvm-project/clang/cmake/modules/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/interpreter/llvm-project/clang/cmake/modules/CMakeLists.txt b/interpreter/llvm-project/clang/cmake/modules/CMakeLists.txt
index 9a3ffb0ee6..d2d6812137 100644
--- a/interpreter/llvm-project/clang/cmake/modules/CMakeLists.txt
+++ b/interpreter/llvm-project/clang/cmake/modules/CMakeLists.txt
@@ -9,7 +9,7 @@ include(FindPrefixFromConfig)
 set(CLANG_INSTALL_PACKAGE_DIR "${CMAKE_INSTALL_PACKAGEDIR}/clang" CACHE STRING
   "Path for CMake subdirectory for Clang (defaults to '${CMAKE_INSTALL_PACKAGEDIR}/clang')")
 # CMAKE_INSTALL_PACKAGEDIR might be absolute, so don't reuse below.
-set(clang_cmake_builddir "${CLANG_BINARY_DIR}/lib${LLVM_LIBDIR_SUFFIX}/cmake/clang")
+set(clang_cmake_builddir "${CMAKE_BINARY_DIR}/lib${LLVM_LIBDIR_SUFFIX}/cmake/clang")
 
 # Keep this in sync with llvm/cmake/CMakeLists.txt!
 set(LLVM_INSTALL_PACKAGE_DIR "${CMAKE_INSTALL_PACKAGEDIR}/llvm" CACHE STRING
-- 
2.43.5

